### PR TITLE
Props Convert

### PR DESCRIPTION
**Pull request by ssorallen**
- Use updated `Polymer.dom` for distributed nodes
- Convert to `properties` object

### CODE REVIEW

File: core-header-panel.html [hunk 1]
```diff
@@ -139,12 +139,24 @@
...
        */
 
-      computed: {
-        dropShadowClass: 'calcDropShadowClass(atTop, mode, shadow)',
-        mainContainerClass: 'calcMainContainerClass(mode)'
-      },
+      properties: {
+
+        animateDuration: {
+          value: 200
```
Comment (cdata) [inline core-header-panel.html]: No `type`?

File: core-header-panel.html [hunk 2]
```diff
@@ -139,12 +139,24 @@
...
+        },
+
+        atTop: {
+          value: true
```
Comment (cdata) [inline core-header-panel.html]: Default value of `true` is not supported for booleans, strictly speaking. Is this an effectively private property?

File: core-header-panel.html [hunk 3]
```diff
@@ -288,7 +300,7 @@
       },
 
       get header() {
-        return this.localDom.distributedNodes(this.$.headerContent)[0];
+        return Polymer.dom.distributedNodes(this.$.headerContent)[0];
```
Comment (cdata) [inline core-header-panel.html]: This API will be different by EOD.

File: core-header-panel.html (hunk 2 above)
Comment (ssorallen) [inline core-header-panel.html]: Yup. By omitting `type` I'm sort of making these "private". I could just assign them in `ready` too?

**Pull request merged**